### DISCUSSION TITLE
Public Access to Multi-step lead form

### DISCUSSION THREAD

**Discussion by andrewijm** [Budibase/budibase#3610] [Help] [answered]
Seriously stupid question here, but I'm testing the Multi-step lead form and can't figure out how to view it as public?

I can access the app as admin user area which is great, but when I try and access the app as public (incognito or logged out), I'm forced to login. What am I missing here?

**Comment by andrewijm** [comment, marked as answer]
Confusion comes from clicking the Play button to use the app. Grabbed link from main app page and works as expected.

**Reply by nunick** [reply]
Hi @andrewijm, were you testing the multi-step lead form from the template on the Budibase website? Or building your own example from scratch and testing it?

I was trying to test out the Multi-step lead form template Budibase offers, but it doesn't seem like the template include the multi-step form anymore. As in, I don't see any screen in the Design page that includes it. Whereas in their August release video they showed the template and it had a screen with the MSLF.

**Reply by joebudi** [reply]
Did you try the public access option in the design section?
If you are logged in you will be authenticated so you can't see it in preview mode. Publish the app, and copy the url. Try pasting the URL into a different browser - brave, or google incognito.

**Reply by nunick** [reply]
@joebudi thank you for the feedback! That worked perfectly. I didn't realize that the design sections were organized this way. 

For others who find this in the future. My issue was solved by selecting "Public" from the "Filter by Access" dropdown in the Design section (previously on Admin) as shown in the image below.

EDIT: And yes, to access the published Public screens, I had to view the app link Incognito/Private mode or a different browser where I wasn't authenticated.

![image](https://user-images.githubusercontent.com/94874984/149640153-56897be1-7136-4c62-97a9-d65c331f08f1.png)

**Reply by joebudi** [reply]
We'll try to improve the experience around this in the future. Stay tuned.
